templates/webapi/search/search.html.ep

```diff
@@ -18,4 +18,4 @@
     </p>
     <div id="results" class="list-group">
     </div>
-</div>
\ No newline at end of file
+</div>
```

Comment:
Looks unintended (but also not really a problem).

Reply:
Indentation is unchanged and matches the opener... so not sure there's an issue here